Adding two traits for S144 (#958)
pi-base/data: spaces/S000144/properties/P000089.md

---
space: S000144
property: P000089
value: true
---

> Note that in $X$, $x \leq y$ iff $x \in \overline{\{y\}}$, i.e., $\leq$ corresponds to the specialization preorder. Now, let $f: X\to X$ be continuous, then we have $x \leq y$ implies $f(x) \leq f(y)$. Hence, if neither $0$ nor $1$ are fixed points, the range of $f$ must be $\{a\}$ or $\{b\}$, making either $a$ or $b$ a fixed point.

Reviewer: ```suggestion
Note that in $X$, $x \leq y$ iff $x \in \overline{\{y\}}$, i.e., $\leq$ corresponds to the specialization preorder, which is preserved by continuous functions. Let $f: X\to X$ be continuous. Then $x \leq y$ implies $f(x) \leq f(y)$. Hence, if neither $0$ nor $1$ are fixed points, the range of $f$ must be $\{a\}$ or $\{b\}$, making either $a$ or $b$ a fixed point.
```
minor tweak, seemed clearer to me
